File: BEE 1038.cpp
#include<iostream>
#include<iomanip>
#define FIXED_FLOAT(x) std::fixed << std::setprecision(2) << (x)

using namespace std;

int main(){
    int code, items;
    cin >> code >> items;
    if (code == 1){
        cout << "Total: R$ " << FIXED_FLOAT(4.0 * items) << endl;
    }
    else if (code == 2){
        cout << "Total: R$ " << FIXED_FLOAT(4.50 * items) << endl;
    }
    else if (code == 3){
        cout << "Total: R$ " << FIXED_FLOAT(5.0 * items) << endl;
    }
    else if (code == 4){
        cout << "Total: R$ " << FIXED_FLOAT(2.0 * items) << endl;
    }
    else if (code == 5){
        cout << "Total: R$ " << FIXED_FLOAT(1.5 * items) << endl;
    }
    return 0;
}

File: BEE 1038.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int code = input.nextInt();
        int items = input.nextInt();
        if (code == 1){
            System.out.printf("Total: R$ %.2f\n", 4.0 * items);
        }
        else if (code == 2){
            System.out.printf("Total: R$ %.2f\n", 4.50 * items);
        }
        else if (code == 3){
            System.out.printf("Total: R$ %.2f\n", 5.0 * items);
        }
        else if (code == 4){
            System.out.printf("Total: R$ %.2f\n", 2.0 * items);
        }
        else if (code == 5){
            System.out.printf("Total: R$ %.2f\n", 1.5 * items);
        }
    }
}